module/Finna/src/Finna/RecordDriver/SolrEad3.php

```diff
@@ -207,66 +219,78 @@ public function getNonPresenterAuthors()
     }
 
     /**
-     * Get location info to be used in LoacationsEad3-record page tab.
```

Comment:
Vaikuttaa voimakkaasti taaksepäin yhteensopimattomalta muutokselta. Onko vaikutukset arvioitu?

Reply:
On yhteensopimaton, mutta Ead3-ajuria ei käytetä vielä tuotannossa.